### ISSUE TITLE
Autoregressive approaches

### ISSUE THREAD

**Issue by ContiPaolo** [alan-turing-institute/autoemulate#831]
Autoregressive (AR) approaches are dominant in temporal models. The major bottleneck of AR strategies is that at testing there is accumulation of error over long rollouts, while we train emulator on ground-truth one-step data, thus the emulator is not trained on data corrupted by this sort of accumulated noise.

Some of the main strategies to deal with this issue are:

1. **Use AR models of order >1**. We don't just train the emulator from one state to the state at the following time step, but we consider a sequence of states as input and we predict as sequence of state in outputs (e.g., we use 10 time steps to predict the following 5). 

> Example: FNO-RNN

2. **Inject noise**. At training we corrupt data with random-walk noise, so the training distribution is closer to the distribution generated during rollouts.

> Example: [Message passing Neural PDE solver](https://arxiv.org/pdf/2202.03376), [Learning to Simulate Complex Physics with Graph Networks](https://arxiv.org/pdf/2002.09405)

3. **Consider multiple AR(1) models**. _n_ models are trained sequentially where the predictions of previous model are given as input to the next model (mimicking regression used at testing). So we can do rollout with lead time _n_

> Example: [Aardvark](https://www.nature.com/articles/s41586-025-08897-0)
4. **AR loss**. We consider a recurrent structure where rollouts are done in the training phase as well and we consider a loss function which accounts for that. This is what Peter presented at the Horizon Scanning
